tests/test_cli.py

```diff
@@ -129,3 +135,19 @@ def test_process(app, names_tar_file, names_service):
 
     assert results.total == 1
     assert list(results.hits)[0]["identifiers"][0]["identifier"] == orcid
+
+
+@patch.object(NamesService, "read")
+@patch.object(NamesService, "update")
+@patch.object(NamesService, "create")
+def test_update_cmd(p_create, p_update, p_read, app, names_tar_file):
+    with app.app_context():
+        runner = CliRunner()
+        result = runner.invoke(
```

Comment:
fails always with:
```
'Usage: vocabularies [OPTIONS] COMMAND [ARGS]...\n\nError: Could not locate a Flask application. You did not provide the "FLASK_APP" environment variable, and a "wsgi.py" or "app.py" module was not found in the current directory.\n'
```

Any ideas? I've checked `invenio-base` tests but do not help much...